src/reducer/notesReducer.js

```diff
@@ -0,0 +1,18 @@
+const notesReducer = (state, action) => {
+    switch (action.type) {
+        case "GET_NOTES":
+            return { ...state, notes: action.payload }
+        case "ADD_NOTE":
+            return { ...state, notes: action.payload }
+        case "REMOVE_NOTE":
+            return { ...state, notes: action.payload }
+        case "UPDATE_NOTE":
+            return { ...state, notes: action.payload }
```

Comment:
Here you are repeating the same code for all the cases, maybe you can create a single case that can be re-used for all the above, that would not violate the DRY principle.

Reply:
Yes, I would have surely done that but I am following this approach so that later on when we would be adding our backend there we might not send the same response every time, so I am having different cases because of that.